pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/exceptions/CircularEvolutionChain.java

```diff
@@ -0,0 +1,27 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.exceptions;
+
+/**
+ * An exception for situations when altering an evolution chain would create
+ * a circular evolution chain.
+ * 
+ * @author Tibor Zauko 433531
+ */
+public class CircularEvolutionChain extends Exception {
```

Comment:
I would rename this to CircularEvolutionChainException - it is more clearer what it is.

Reply:
Done